primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -92,8 +92,9 @@ impl<E: Pairing> PolynomialCommitmentScheme for UnivariateKzgPCS<E> {
         poly: &Self::Polynomial,
     ) -> Result<Self::Commitment, PCSError> {
         let prover_param = prover_param.borrow();
-        let commit_time =
-            start_timer!(|| format!("Committing to polynomial of degree {} ", poly.degree()));
+        // let commit_time =
```

Comment:
Minor: we can uncomment it but add a flag-feature so that it doesn't execute when we do disperse benchmark.

Reply:
Oops! Forgot to uncomment this. Add `kzg-print-trace` feature flag in https://github.com/EspressoSystems/jellyfish/pull/389/commits/a0678336d0061f4f6fb2406401a5cc2179191c2e . I added it only to those timers that were causing me noise. I could add it everywhere else, too, but it does pollute the code.